addons/op-badge/userscript.js

```diff
@@ -0,0 +1,17 @@
+export default async function({ addon, console }) {
+    const posts = document.querySelectorAll(".blockpost");
+
+    const op = await fetch(`https://scratchdb.lefty.one/v3/forum/topic/posts/${window.location.href.split("/")[5]}/0?o=oldest`).then(res => res.json());
+
+    for (const i of posts) {
+        if (i.querySelector(".username").innerText == op[0].username) {
```

Comment:
This won't work for Scratch Team members since their username will end with a `star*`

Reply:
It displays the scratch team usernames normally on ScratchDB...

Reply:
> It displays the scratch team usernames normally on ScratchDB...

Sure, but `i.querySelector(".username").innerText` will include the asterisk, so both strings won't be equal.
This is just an edge-case that I think was worth mentioning, since I wasn't sure someone else would bring it up.

Reply:
You're right, unlike profile pages, an asterisk isn't added in the forums. My bad.